apps/extension/src/Approvals/ConfirmSignLedgerTx.tsx

```diff
@@ -125,10 +125,12 @@ export const ConfirmSignLedgerTx: React.FC<Props> = ({ details }) => {
         if (!accountDetails) {
           throw new Error(`Failed to query account details for ${signer}`);
         }
-        const path = makeBip44Path(
-          chains.namada.bip44.coinType,
-          accountDetails.path
-        );
+        const path = {
+          account: accountDetails.path.account,
+          change: accountDetails.path.change || 0,
+          index: accountDetails.path.index || 0,
+        };
```

Comment:
For signing with transparent keys, all of these values are expected to be present!